Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -283,6 +302,92 @@ namespace AzToolsFramework::Prefab
         return GetInstanceReference(m_rootAliasFocusPath);
     }
 
+    LinkId PrefabFocusHandler::AppendPathFromFocusedInstanceToPatchPaths(PrefabDom& providedPatch, const AZ::EntityId& entityId) const
```

Comment:
I don't see calls to this function, which makes it hard to judge whether this function belongs in this class or what changes need to be done to it.

Maybe this can be removed for now and added later with its usage?